src/app/payment/paymentMethod/MasterpassPaymentMethod.tsx

```diff
@@ -20,8 +20,17 @@ const MasterpassPaymentMethod: FunctionComponent<MasterpassPaymentMethodProps &
         },
     }), [initializePayment]);
 
+    const formatLocale = useCallback((localeLanguage: string) => {
+        const auxLocale = localeLanguage.replace(/[-_]/g, '_');
+        const formatedLocale = auxLocale.includes('_') ? `${auxLocale}` : `${auxLocale}_${auxLocale}`;
```

Comment:
There's no need to wrap the first part of the ternary in a string literal, `auxLocale` is already a string.

```suggestion
        const formatedLocale = auxLocale.includes('_') ? auxLocale : `${auxLocale}_${auxLocale}`;
```